src/cmd/flux-resource.py

```diff
@@ -344,11 +394,9 @@ def status(args):
 
 
 def drain_list(args):
+    fmt = FluxResourceConfig("drain").load().get_format_string(args.format)
     args.from_stdin = False
-    args.no_header = False
```

Comment:
nit: the removal of `args.no_header = False` in ` flux-resource: support named formats via UtilConfig ` probably should have been in a prior commit.  Maybe in the commit that added `print_items()` b/c that took `no_header=args.no_header`?

Reply:
This might have been a bit confusing -- `args.no_header` is removed here because a `--no-header` option was added to `flux resource drain` in this commit, so we no longer have to set it in the `Namespace` object before calling `status()`.